package/scripts/postinstall

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Would be good to document e.g. expected arguments etc. and fail if they aren't passed.

Reply:
We only really care about the tmp install in `$2` and that's already failing if missing. So this would be enough?
```suggestion
#!/bin/bash
# $1 Full path to the installer - unused
# $2 Location of the temporary brew install we're moving into place
# $3 Target install location - unused
# $4 System root directory - unused
```

Reply:
Stupid question: why have 1/3/4 if we don't care about them?

Reply:
Or is it some fixed API that we're following?

Reply:
Yeah, this is what the macOS system passes to all post/pre-install scripts

Reply:
Ah gotcha.

Reply:
Yeh, in that case 👍🏻 to the comment you have here.